test/tools/unified-spec-runner/operations.ts

```diff
@@ -741,9 +790,10 @@ export async function executeOperationAndCheck(
   const opFunc = operations.get(operation.name);
   expect(opFunc, `Unknown operation: ${operation.name}`).to.exist;
 
-  if (operation.arguments?.session) {
-    const session = entities.getEntity('session', operation.arguments.session, false);
-    operation.arguments.session = session;
+  if (typeof operation.arguments?.session === 'string') {
+    // Cannot do this b/c operations pass through unsanitized options to command construction:
```

Comment:
for debugging, since we get the session here I was not able to see what the session's name was inside the operation function, so I was attaching the name as an underscore prop just to see the name. Turns out that will make some operations fail b/c it will take the options and directly map them to the command construction. Just sharing context, removed the comment.